qiskit/ignis/verification/randomized_benchmarking/interleaved_fitters.py

```diff
@@ -0,0 +1,274 @@
+# -*- coding: utf-8 -*-
```

Comment:
This shouldn't be in a separate file, should be in fitters.py

Reply:
Done.